docs/logging-usage.rst

```diff
@@ -392,70 +392,105 @@ Delete a sink:
 Integration with Python logging module
 ---------------------------------------------
 
-
-It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
-create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
-Logging client.
+It's possible to tie the Python :mod:`logging` module directly into Google Stackdriver Logging.
+There are different handler options to accomplish this. To automatically pick the default,
+use :meth:``google.cloud.logging.client.get_default_handler`.
```

Comment:
Jinx on the backtick. Also suggest using a tilde to make the docs easier on the eyes

```
:meth:`~google.cloud.logging.client.get_default_handler`
```